mmdet3d/apis/inference.py

```diff
@@ -100,15 +119,16 @@ def show_result_meshlab(data, result, out_dir):
 
     assert out_dir is not None, 'Expect out_dir, got none.'
 
-    pred_bboxes = result[0]['boxes_3d'].tensor.numpy()
+    if 'pts_bbox' in result[0].keys():
+        pred_bboxes = result[0]['pts_bbox']['boxes_3d'].tensor.numpy()
+    else:
+        pred_bboxes = result[0]['boxes_3d'].tensor.numpy()
     # for now we convert points into depth mode
     if data['img_metas'][0][0]['box_mode_3d'] != Box3DMode.DEPTH:
         points = points[..., [1, 0, 2]]
         points[..., 0] *= -1
         pred_bboxes = Box3DMode.convert(pred_bboxes,
                                         data['img_metas'][0][0]['box_mode_3d'],
                                         Box3DMode.DEPTH)
-        pred_bboxes[..., 2] += pred_bboxes[..., 5] / 2
```

Comment:
This changes the coordinates of boxes, check whether they are necessary.

Reply:
it is unnecessary, because now this ops is done in this [line](https://github.com/open-mmlab/mmdetection3d/blob/22c47ac75781b4a256cdde1074b7c2bb207b2479/mmdet3d/core/visualizer/show_result.py#L100)